Mobile Test Runner for Jama/Mobile Test Runner for Jama.xcodeproj/project.pbxproj

```diff
@@ -337,7 +340,7 @@
 				TargetAttributes = {
 					13A342EA1F0367E9002EBC92 = {
 						CreatedOnToolsVersion = 8.3.3;
-						DevelopmentTeam = 5BL5M2XBM2;
+						DevelopmentTeam = LA4QFXA5C5;
```

Comment:
I know this is a generated file but else where it appears the development team was changed